pyaraucaria/airmass_calculator.py

```diff
@@ -0,0 +1,11 @@
+import math
+
+class AirmassCalculator:
+    @staticmethod
```

Comment:
The class with single static method is pointless. I would remove it and leave this method only as a function.